{
  "errors": [
    {
      "code": 2,
      "level": "warn",
      "message": "[WARN] Semgrep Core \u2014 Timeout\nAn error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep\n\nWhen running rules.forcetimeout on targets/equivalence/open_redirect.py: \n",
      "path": "targets/equivalence/open_redirect.py",
      "rule_id": "rules.forcetimeout",
      "type": "Timeout"
    }
  ],
  "results": []
}